hubble/hubble.go

```diff
@@ -237,16 +246,40 @@ func (k *K8sHubble) Enable(ctx context.Context) error {
 		return err
 	}
 
+	warningFreePods := []string{defaults.AgentDaemonSetNae}
 	if k.params.Relay {
 		if err := k.enableRelay(ctx); err != nil {
 			return err
 		}
+		warningFreePods = append(warningFreePods, defaults.RelayDeploymentName)
 	}
 
 	if k.params.UI {
 		if err := k.enableUI(ctx); err != nil {
 			return err
 		}
+		warningFreePods = append(warningFreePods, defaults.HubbleUIServiceName)
```

Comment:
I'm unsure whether that is working as it seems to me that [status is currently lacking support for Hubble UI](https://github.com/cilium/cilium-cli/blob/master/status/k8s.go#L287).

Reply:
Hey, I have added the support for Hubble UI. Can you check it out and let me know.